### DISCUSSION TITLE
any plans for a discord?

### DISCUSSION THREAD

**Discussion by avonian** [connery-io/connery-sdk#108] [Q&A] [answered]
wondering if you guys have plans for a discord or slack? if so, would love to join, as it might make it easier for some of us early adopters/potential-contributors :)

**Comment by machulav** [comment, marked as answer]
@avonian here it is: https://discord.gg/d45vsC6Z - you will be the first! 🙂

We actually created the Discord server some time ago but never set it up properly and promoted it.
I will add it to the repo's README now, as it makes total sense!  Thanks for the suggestion!
